integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java

```diff
@@ -217,6 +217,7 @@
   public static final String MII_BASIC_APP_NAME = "sample-app";
   public static final String MII_BASIC_APP_DEPLOYMENT_NAME = "myear";
   public static final String MII_TWO_APP_WDT_MODEL_FILE = "model-singlecluster-two-sampleapp-wls.yaml";
+  public static final String MII_UPDATED_RESTART_REQUIRED_LABEL = "weblogic.configurationChangedPendingRestart";
```

Comment:
Tom requested changing the label and now the label is 'weblogic.configChangesPendingRestart'

Reply:
Done